libs/pages/dashboard/src/lib/ui/manage-removed-items/manage-removed-items.component.html

```diff
@@ -0,0 +1,49 @@
+<button
+  (click)="openModal()"
+  class="btn btn-primary btn-outline"
+  type="button"
```

Comment:
```suggestion
  type="button"
  [disabled]="items.length === 0"
```

Reply:
It may need the use of observables to work without reloading.

Reply:
I juste remove the change detection because I need to learn more about why sometime it does not work and I need to understand how to build component arround this.